src/platform/intel/cavs/platform.c

```diff
@@ -432,15 +427,15 @@ int platform_init(struct sof *sof)
 	pm_runtime_get(PM_RUNTIME_DSP, PWRD_BY_HPRO | (PLATFORM_CORE_COUNT - 1));
 #endif
 	/* request configured ring oscillator and wait for status ready */
-	shim_write(SHIM_CLKCTL, shim_read(SHIM_CLKCTL) | CAVS_DEFAULT_RO);
-	while (!(shim_read(SHIM_CLKSTS) & CAVS_DEFAULT_RO))
+	shim_write(SHIM_CLKCTL, shim_read(SHIM_CLKCTL) | CAVS_ROS);
+	while (!((shim_read(SHIM_CLKSTS) & CAVS_ROS) == CAVS_ROS))
 		idelay(16);
 
 	shim_write(SHIM_CLKCTL,
-		   CAVS_DEFAULT_RO | /* Request configured RING Osc */
-		   CAVS_DEFAULT_RO_FOR_MEM | /* Select configured
-					     * RING Oscillator Clk for memory
-					     */
+		   CAVS_ROS | /* Request configured RING Osc */
+		   CAVS_ROS_FOR_MEM | /* Select configured
```

Comment:
Could you explain why single RO selection is changed to selecting both ROs for memory?

Reply:
It was only a temp to combine the configuration of the DEFAULT RO for both CAVS_LPRO selected and deselected.